platform-emf/src/main/java/com/softicar/platform/emf/form/scope/EmfFormViewScopeAction.java

```diff
@@ -0,0 +1,90 @@
+package com.softicar.platform.emf.form.scope;
+
+import com.softicar.platform.common.core.i18n.IDisplayString;
+import com.softicar.platform.common.core.utils.CastUtils;
+import com.softicar.platform.common.io.resource.IResource;
+import com.softicar.platform.db.sql.field.ISqlForeignRowField;
+import com.softicar.platform.dom.elements.popup.manager.DomPopupManager;
+import com.softicar.platform.emf.EmfI18n;
+import com.softicar.platform.emf.EmfImages;
+import com.softicar.platform.emf.action.IEmfCommonAction;
+import com.softicar.platform.emf.attribute.field.foreign.row.IEmfForeignRowAttribute;
+import com.softicar.platform.emf.form.IEmfFormBody;
+import com.softicar.platform.emf.form.popup.EmfFormPopup;
+import com.softicar.platform.emf.mapper.IEmfTableRowMapper;
+import com.softicar.platform.emf.permission.EmfMappedPermission;
+import com.softicar.platform.emf.permission.EmfPermissions;
+import com.softicar.platform.emf.permission.IEmfPermission;
+import com.softicar.platform.emf.predicate.EmfPredicate;
+import com.softicar.platform.emf.predicate.IEmfPredicate;
+import com.softicar.platform.emf.table.IEmfTable;
+import com.softicar.platform.emf.table.row.IEmfTableRow;
+
+public class EmfFormViewScopeAction<R extends IEmfTableRow<R, ?>, F extends IEmfTableRow<F, ?>> implements IEmfCommonAction<R> {
+
+	private final IEmfTable<R, ?, ?> table;
+	private final IEmfTable<F, ?, ?> scopeTable;
+
+	public EmfFormViewScopeAction(IEmfTable<R, ?, ?> table) {
+
+		this.table = table;
+		this.scopeTable = table//
+			.getScopeField()
+			.map(this::getScopeTable)
+			.orElse(null);
+	}
+
+	@Override
+	public IResource getIcon() {
+
+		return EmfImages.VIEW_SCOPE.getResource();
+	}
+
+	@Override
+	public IDisplayString getTitle() {
+
+		IDisplayString title = scopeTable != null? scopeTable.getTitle() : IDisplayString.EMPTY;
```

Comment:
I think we should create in `EmfI18n`:
```
I18n0 VIEW_SCOPE = new I18n0("View Scope")//
		.de("Scope ansehen");
```
Otherwise, this action would have an empty name in the EMF table analysis pages of the core module, which would be confusing.